contracts/apps/disputable/DisputableAragonApp.sol

```diff
@@ -138,7 +138,11 @@ contract DisputableAragonApp is IDisputable, AragonApp {
     */
     function _closeAgreementAction(uint256 _actionId) internal {
         IAgreement agreement = _ensureAgreement();
-        agreement.closeAction(_actionId);
+        (,,,,, bool closed,,) = agreement.getAction(_actionId);
+
+        if (!closed) {
+            agreement.closeAction(_actionId);
```

Comment:
Not opposed to it, but I think we should do a small trial run to see how much the gas difference actually is, if this is just an optimization.

The extra call to `agreement.getAction()` may remove the large part of the savings if `closeAction()` is fairly simple on already-closed actions.

Reply:
Yea! I tested this and it is not an optimizing it actually, I'll roll it back